php-practice/php-practice_2.php

```diff
@@ -1,15 +1,143 @@
 <?php
 // Q1 tic-tac問題
+for ($i = 1; $i <=  100; $i++) {
+  if ($i % 4 == 0 && $i % 5 == 0) {
+      echo 'tic-tac' . "\r\n";
+  } elseif ($i % 4 == 0) {
+      echo 'tic' . "\r\n";
+  } elseif ($i % 5 == 0) {
+      echo 'tac' . "\r\n";
+  } else {
+      echo $i . "\r\n";
+  }
+}
 
 
 // Q2 多次元連想配列
+// 問題1
+$personalInfos = [
+  [
+      'name' => 'Aさん',
+      'mail' => 'aaa@mail.com',
+      'tel'  => '09011112222'
+  ],
+  [
+      'name' => 'Bさん',
+      'mail' => 'bbb@mail.com',
+      'tel'  => '08033334444'
+  ],
+  [
+      'name' => 'Cさん',
+      'mail' => 'ccc@mail.com',
+      'tel'  => '09055556666'
+  ],
+];
+echo $personalInfos[1]['name'] . 'さんの電話番号は' . $personalInfos[1]['tel'] . 'です。';
 
+//問題２
+$personalInfos = [
+  [
+      'name' => 'Aさん',
+      'mail' => 'aaa@mail.com',
+      'tel'  => '09011112222'
+  ],
+  [
+      'name' => 'Bさん',
+      'mail' => 'bbb@mail.com',
+      'tel'  => '08033334444'
+  ],
+  [
+      'name' => 'Cさん',
+      'mail' => 'ccc@mail.com',
+      'tel'  => '09055556666'
+  ],
+];
+foreach ($personalInfos as $key => $personalInfos){
```

Comment:
[nits]
`{`の前後は半角スペースを設けてください！
```suggestion
foreach ($personalInfos as $key => $personalInfos) {
```